### ISSUE TITLE
2 permissions for noburn

### ISSUE THREAD

**Issue by johncsuti** [BentoBoxWorld/AcidIsland#81]
**Description**
2 versions of no burn permission? There should only be one either admin or mod not both.

**Steps to reproduce the behavior:**
N/A

**Expected behavior**
1 permission per item either acidisland.admin.noburn or acidisland.mod.noburn not both.

**Screenshots**
N/A

**Server Information:**

[Please complete the following information:]
N/A

**Additional context**
N/A

**Comment by tastybento** [comment]
The docs are wrong - there's no `acidisland.admin.noburn` permission. There was an `admin.noburn` check, but I'm going to remove it.

**Comment by johncsuti** [comment]
@tastybento The wiki was wrong by my revision. I looked through the code and noticed a check for both. I then added them both to the wiki without thinking about it first. My train of thought was mods got no burn on there island admins on all islands.
![Screenshot_2020-02-13 BentoBoxWorld AcidIsland](https://user-images.githubusercontent.com/22971404/74451752-1a11a100-4e4e-11ea-924f-2e725ad34fba.png)
